support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.tsx

```diff
@@ -222,6 +227,9 @@ function withProps(props: PropTypes) {
 	);
 	const lastOneOffContribution = useLastOneOffContribution(props.isSignedIn);
 
+	const showBenefitsMessage = props.sfdv2AbTest === 'SINGLE_FRONT_DOOR_V2';
```

Comment:
Why not just map this to a boolean in `mapStateToProps`?

Reply:
Done a bit of refactoring here to clean this up!